data/remnant/cludge blaster mission.txt

```diff
@@ -30,10 +30,10 @@ mission "Remnant Cludge Blaster Sale"
 					decline
 					
 			label sale
-			`	"I'm sure you'll find the modifications fulfill roles not easily found in smaller weapons," the leading Remnant replies. "For you, we will give four for a million credits. Is that acceptable?" he asks.`
+			`	"I'm sure you'll find the modifications fulfil roles not easily found in smaller weapons," the leading Remnant replies. "For you, we will give four for a million credits. Is that acceptable?"`
 			choice
 				`	"Sounds like a fair deal."`
-				`	"If it'll support the Remnant's future, I might as well."`
+				`	"If it'll support the Remnant future, I might as well."`
```

Comment:
Not sure about this one

Reply:
```suggestion
				`	"If it'll support the future of the Remnant, I might as well."`
```

Reply:
"the future of the Remnant" works, but "the Remnant's future" is again more concise and fits with established use of the polity's name.

Reply:
Either works but the less wordy option is probably better.